### PR TITLE
Update usage for Private devices

### PR DESCRIPTION
**Pull request by adam-saucelabs**
### Description
Updating usage.md for private device usage that we are launching this week 

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/insights/usage-report.md [hunk 1]
```diff
@@ -42,14 +42,25 @@ Shows the total concurrency, aggregating all devices grouped by team. This infor
 - VM Concurrency: Shows the concurrent usage for tests executed on Linux Browsers, Windows Browsers and Android Emulators.
 - Mac VM Concurrency: Shows the concurrent usage for tests executed on Mac OS, Mac OSX and iOS Emulators.
 
-### Real Device Concurrency Usage
+### Real Device Concurrency and Usage
 
-:::note Public and Private Real Devices
-Public Device usage is generally available. Private Device usage will be generally available May 2024
-:::
+With Suace Labs Real Device usage analytics your organization can get a view of both Public device concurrency usage as well as Private device usage to get the most of your Sauce Labs testing strategy. Our real dvice usage analytics give you many of the same benefits as our Virtual device usage reporting such as:
```
Comment (CharlieBurnett) [inline docs/insights/usage-report.md:47]: Spell our company name correctly plz

File: docs/insights/usage-report.md [hunk 2]
```diff
@@ -42,14 +42,25 @@ Shows the total concurrency, aggregating all devices grouped by team. This infor
...
+- Ability to visualize usage at organization level or by team level.
+- Access to expanded date range data and granularity options.
+- Comparison of usage between teams.
+- Visualize the maximum Public device usage as it approaches the contractual limit.
+- Visualize Private device usage to best utilize your organization's devices.
+- Access to usage data via the Sauce Labs UI and API.
 
+ #### Public Device Concurrency
+ 
 <img src={useBaseUrl('img/insights/ccy-org-month-rdc-public.png')} alt="Public Real Device concurrency usage at organization level"/>
```
Comment (CharlieBurnett) [inline docs/insights/usage-report.md:58]: nit: why is Per Month not capitalized in the title but all other words are

**Review by CharlieBurnett — CHANGES_REQUESTED** [review]
Almost perfect, just fix some typoes and formatting

File: docs/insights/usage-report.md (hunk 1 above)
Comment (adam-saucelabs) [reply docs/insights/usage-report.md]: fixed

**Review by CharlieBurnett — APPROVED** [review]

**Pull request merged**